internal/graph/check.go

```diff
@@ -508,16 +508,10 @@ func (c *LocalChecker) dispatch(_ context.Context, req *ResolveCheckRequest) Che
 	}
 }
 
-// ResolveCheck resolves a node out of a tree of evaluations. If the depth of the tree has gotten too large,
-// evaluation is aborted and an error is returned. The depth is NOT increased on computed usersets.
```

Comment:
> The depth is NOT increased on computed usersets.

it is, see how in this test we expect an error: https://github.com/openfga/openfga/blob/1843929c1e76af581754ffe65bd244316c477b82/assets/tests/consolidated_1_1_tests.yaml#L5067-L5110